src/src/PluginStructs/P025_data_struct.cpp

```diff
@@ -17,24 +17,55 @@ int16_t P025_data_struct::read() {
   config |= (0x8000); // Start a single conversion
 
   Wire.beginTransmission(i2cAddress);
-  Wire.write((uint8_t)(0x01));
+  Wire.write((uint8_t)(0x01));   // access to config register
   Wire.write((uint8_t)(config >> 8));
   Wire.write((uint8_t)(config & 0xFF));
   Wire.endTransmission();
 
-  delay(9); // See https://github.com/letscontrolit/ESPEasy/issues/3159#issuecomment-660546091
-  return readRegister025(0x00);
+  while (isReady025()==false) delay(1);
```

Comment:
Comparing to `true` or `false` always looks a bit awkward in C-like programming languages, but not taking care of a timely exit condition is more important, so that should at least be added here.